index.js

```diff
@@ -19,49 +20,62 @@ const SUBMISSION = [
 ]
 
 app.post('/signup', function(req, res) {
-  // Add logic to decode body
-  // body should have email and password
-
-
-  //Store email and password (as is for now) in the USERS array above (only if the user with the given email doesnt exist)
-
-
-  // return back 200 status code to the client
-  res.send('Hello World!')
+ 
+    const { email, password } = req.body;
```

Comment:
@yashhhh04 It's good to use a code formatter so that all your formatting will taken care. (if you are using vscode you can try using prettier)